### PR TITLE
RSpec: split up image_1_spec.rb

### PR DESCRIPTION
**Pull request by mockdeep**

### CODE REVIEW

File: spec/rmagick/image/capture_spec.rb [hunk 1]
```diff
@@ -0,0 +1,15 @@
+RSpec.describe Magick::Image, "#capture" do
+  before do
+    @img = Magick::Image.new(20, 20)
+  end
+
+  it "works" do
+    # expect { Magick::Image.capture }.not_to raise_error
```
Comment (dlemstra) [inline spec/rmagick/image/capture_spec.rb]: Why are these commented out? If they already were can we remove it?

File: spec/rmagick/image/compare_channel_spec.rb [hunk 2]
```diff
@@ -0,0 +1,33 @@
+RSpec.describe Magick::Image, "#compare_channel" do
+  before do
+    @img = Magick::Image.new(20, 20)
+  end
+
+  it "works" do
+    img1 = Magick::Image.read(IMAGES_DIR + '/Button_0.gif').first
```
Comment (dlemstra) [inline spec/rmagick/image/compare_channel_spec.rb]: Not for this PR but maybe we could put `IMAGES_DIR + '/Button_0.gif'` into a `TestImages` class and add `Button0` field/property?

File: spec/rmagick/image/capture_spec.rb (hunk 1 above)
Comment (mockdeep) [reply spec/rmagick/image/capture_spec.rb]: Hmm, not sure why they were commented out, but it was in the original file. I guess I can remove them as they don't appear to be that important.

File: spec/rmagick/image/compare_channel_spec.rb (hunk 2 above)
Comment (mockdeep) [reply spec/rmagick/image/compare_channel_spec.rb]: Yeah, I think we can come up with a better way of handling this.

File: spec/rmagick/image/capture_spec.rb (hunk 1 above)
Comment (mockdeep) [reply spec/rmagick/image/capture_spec.rb]: Alright, updated.

**Pull request merged**